src/picongpu/include/particles/Particles.tpp

```diff
@@ -167,13 +167,19 @@ void Particles<T_ParticleDescription>::update(uint32_t )
         typename GetFlagType<FrameType,interpolation<> >::type
         >::type InterpolationScheme;
 
-    typedef typename GetMargin<InterpolationScheme>::LowerMargin LowerMargin;
-    typedef typename GetMargin<InterpolationScheme>::UpperMargin UpperMargin;
+    typedef typename GetMargin<InterpolationScheme>::LowerMargin InterpolationLowerMargin;
+    typedef typename GetMargin<InterpolationScheme>::UpperMargin InterpolationUpperMargin;
+    typedef typename GetMargin<ParticlePush>::LowerMargin ExtendLowerMargin;
+    typedef typename GetMargin<ParticlePush>::UpperMargin ExtendUpperMargin;
 
     typedef PushParticlePerFrame<ParticlePush, MappingDesc::SuperCellSize,
         InterpolationScheme,
         fieldSolver::NumericalCellType > FrameSolver;
 
+    // adjust interpolation area in particle pusher to allow sub-sampling pushes
+    typedef typename PMacc::math::CT::add< InterpolationLowerMargin, ExtendLowerMargin >::type LowerMargin;
```

Comment:
currently there is no way around to call the margins for the pusher inside FieldB and FieldE.